### ISSUE TITLE
Random failure due to "unreliable" results in emulation test

### ISSUE THREAD

**Issue by AnnaKwa** [ai2cm/fv3net#1469]
I had a random intermittent failure on CircleCI for `external/fv3fit/tests/emulation/thermobasis/test_emulator.py[test_Config_multi_output_levels]`:

```
E   hypothesis.errors.Flaky: Hypothesis test_Config_multi_output_levels(levels=[32007, 7175298582426376939, 117, 29170, 4235]) produces unreliable results: Falsified on the first call but did not on a subsequent one
```

When I reran without changing anything, the test passed. Has anyone run into this before or is it a super rare incident? @nbren12 @frodre 

https://app.circleci.com/pipelines/github/ai2cm/fv3net/10033/workflows/d11d7bcd-88b5-4e2f-a076-fe0bf8e1ad53/jobs/87314

**Comment by nbren12** [comment]
hypothesis does some randomization. Those number of levels seem pretty out
of control though. Surprised it doesn't run out of memory. Perhaps we
should limit the levels to lie within a smaller range like 1 to 100.

**Comment by oliverwm1** [comment]
I think i still got hit by this here:

https://app.circleci.com/pipelines/github/ai2cm/fv3net/10087/workflows/e3aeced6-7cae-4844-a6a2-ed1542c28b7b/jobs/87844

I had merged in master after the fix PR was merged.

**Comment by oliverwm1** [comment]
Hopefully closed by #1546
